HISTORY.rst

```diff
@@ -2,6 +2,26 @@
 History
 =======
 
+v3.45.0 (2021-09-30)
+--------------------
+
+Critical Changes
+
+-  Updated the ``deploy_marketing_cloud_package`` task for compatibility with the October 2021 release of Marketing Cloud. (#2899)
```

Comment:
this should be under changes, not critical changes. the only change was adding support for a new return status, so it's backwards compatible